app/AlexNet/CMakeLists.txt

```diff
@@ -1,36 +1,32 @@
-set(INCLUDE_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/alexnet.hpp")
-
-set(SRC_FILES "${CMAKE_SOURCE_DIR}/src/alexnet/alexnet_src.cpp")
+file(GLOB_RECURSE SRC_FILES *.cpp)
+file(GLOB_RECURSE INCLUDE_HEADERS *.h *.hpp)
 
+set(INCLUDE_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/alexnet.hpp")
+set(SRC_FILES "${CMAKE_CURRENT_SOURCE_DIR}/alexnet_src.cpp")
 add_library(AlexNetLib STATIC ${INCLUDE_HEADERS} ${SRC_FILES})
 
 if(UNIX)
   unset(TENSORFLOW_FOUND)
-
   find_path(tensorflow_INCLUDE_DIR
           NAMES
           tensorflow/core
           tensorflow/c
           HINTS
           /usr/local/include/tensorflow
           /usr/include/tensorflow)
-
   find_library(tensorflow_LIBRARY NAMES tensorflow
           HINTS
           /usr/lib
           /usr/local/lib)
-
   include(FindPackageHandleStandardArgs)
   find_package_handle_standard_args(tensorflow DEFAULT_MSG tensorflow_INCLUDE_DIR tensorflow_LIBRARY)
-
   if(TENSORFLOW_FOUND)
```

Comment:
If Tensorflow is not found, please do not build this application and print a warning

Reply:
only on linux?

Reply:
On any platform

Reply:
Currently it still behaves the same way in case Tensorflow is not installed. We need to print a warning that Tensorflow is not found and do not build this target (do not initialize target, it's include and link flags and so on)

Reply:
Now I understand what you mean